app/core/views/cities/index.pug

```diff
@@ -1,9 +1,21 @@
-include ../_components/html
-
-+doc()
-  +head()
-  +body()
+doctype html
+html(lang='en')
+  head
+    meta(charset='UTF-8')
+    meta(http-equiv='X-UA-Compatible', content='IE=Edge')
+    meta(content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name='viewport')
+    link(href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet')
+    link(href='/css/home.min.css', rel='stylesheet')
+    link(href='/css/form.min.css', rel='stylesheet')
+    script(src='https://code.jquery.com/jquery-3.3.1.min.js')
```

Comment:
`script` tags should not be placed in the head, but rather in the body, at the end of the closing tag.